howtos/how-to-create-a-poll/server.js

```diff
@@ -0,0 +1,170 @@
+import express from "express";
+import fetch from "node-fetch";
+import cors from "cors";
+
+// Replace with your app ID and secret key
+const appId = "<APP_ID>";
+const secretKey = "<SECRET_KEY>";
+
+const basePath = "https://api.talkjs.com";
+
+const app = express();
+app.use(express.static('public'));
+app.use(cors());
+app.use(express.json());
+
+app.listen(3000, () => console.log("Server is up"));
+
+async function deleteSystemMessage(messageId, conversationId) {
+  console.log("Deleting message with id:", messageId);
+  return fetch(`${basePath}/v1/${appId}/conversations/${conversationId}/messages/${messageId}`, {
+    method: "DELETE",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${secretKey}`,
+    },
+  });
+}
+
+app.post("/deleteSystemMessage", async (req, res) => {
+  const { messageId, conversationId } = req.body;
+  await deleteSystemMessage(messageId, conversationId);
+  res.status(200).end();
+});
+
+// EVERYTHING BELOW IS SETUP CODE FOR THIS EXAMPLE
```

Comment:
Would it maybe make sense to move this out into a separate JavaScript file that people can just run? Like, if you call it `setup.js`, then people can just call `node setup.js` in this hotwo's folder, and have it set up.

Reply:
Right now users run `npm start` and everything is set up thanks to the following in package.json "scripts": {
    "start": "node server.js"
  },
  
 If we moved this code to a different file, I'm not sure it would be as clean as the current implementation. However, my knowledge of node is very basic. 
  
  @keerlu Do you this would be a help or a hindrance to the user in terms of them getting set up and running the tutorial quickly?

Reply:
Hm, I don't have a strong opinion, it'd be conceptually cleaner to have it in a separate file to the server code but it's also quite nice to just have everything in one file to run with one command.

I think it's fine to leave as is for this tutorial, but I'll think about doing that for future ones.

Reply:
Yeah, I definitely see your point about the ease of getting it up and running quickly. To me it seemed like it would make sense to separate it out, so you could run them separately, without re-creating the demo data on every run of the server, but maybe that doesn't matter that much for a sample app.